TestFoundation/TestPipe.swift

```diff
@@ -35,20 +35,19 @@ class TestPipe: XCTestCase {
         pipes = []
     }
 
-    func test_Pipe() {
+    func test_Pipe() throws {
```

Comment:
Doesn't this disqualify the test from XCTest (acknowledging that we do not automatically discover them at this time).

Reply:
No; this is a pattern we use elsewhere that XCTest supports.